llms/mlx_lm/utils.py

```diff
@@ -220,11 +247,16 @@ def logit_bias_processor(_, logits):
 
     # Create the KV cache for generation
     if prompt_cache is None:
-        prompt_cache = cache.make_prompt_cache(model, max_kv_size)
+        prompt_cache = cache.make_prompt_cache(
```

Comment:
Nitpick: There are a few things that have been split into multiple lines and I wonder if some config is different on your machine.